docs/output-files/dat-hammy-traces.md

```diff
@@ -2,26 +2,76 @@
 layout: default
 title: (*.dat) HaMMy traces
 parent: /output-files.html
-nav_order: 1
+nav_order: 7
 nav_exclude: 1
 ---
 
```

Comment:
There are sections for the .mash file and the .sira files.
Pre-set parameters are regular .mat files (not specific to MASH) and are detailed in the section `Simulation / Panels/ Molecules`.
Then .sif files are from Andor Solis and .tif is a standard, writing sections in MASH documentation is kind of weird for me.. It would be like writing a PNG section.